integration/tests/posit/connect/test_bundles.py

```diff
@@ -0,0 +1,99 @@
+import time
+from pathlib import Path
+
+import pytest
+from packaging import version
+
+from posit import connect
+
+from . import CONNECT_VERSION
+
+
+class TestBundles:
+    @classmethod
+    def setup_class(cls):
+        cls.client = connect.Client()
+        cls.content = cls.client.content.create(
+            name=f"test-bundles-{int(time.time())}",
+            title="Test Bundles",
+            access_type="all",
+        )
+        # Path to the test bundle
+        bundle_path = Path(
+            "../../../resources/connect/bundles/example-flask-minimal/bundle.tar.gz"
+        )
+        cls.bundle_path = (Path(__file__).parent / bundle_path).resolve()
+
+    @classmethod
+    def teardown_class(cls):
+        cls.content.delete()
+
+    def test_create_bundle(self):
+        """Test creating a bundle."""
+        bundle = self.content.bundles.create(str(self.bundle_path))
+        assert bundle["id"]
+        assert bundle["content_guid"] == self.content["guid"]
+
+    def test_find_bundles(self):
+        """Test finding all bundles."""
+        # Create a bundle first
+        self.content.bundles.create(str(self.bundle_path))
+
+        # Find all bundles
+        bundles = self.content.bundles.find()
+        assert len(bundles) >= 1
+
+    def test_find_one_bundle(self):
+        """Test finding a single bundle."""
+        # Create a bundle first
+        self.content.bundles.create(str(self.bundle_path))
+
+        # Find one bundle
+        bundle = self.content.bundles.find_one()
+        assert bundle is not None
+        assert bundle["content_guid"] == self.content["guid"]
+
+    def test_get_bundle(self):
+        """Test getting a specific bundle."""
+        # Create a bundle first
+        created_bundle = self.content.bundles.create(str(self.bundle_path))
+
+        # Get the bundle by ID
+        bundle = self.content.bundles.get(created_bundle["id"])
+        assert bundle["id"] == created_bundle["id"]
```

Comment:
These were all tests for functionality we had already but was not (yet) tested, yeah?

Reply:
Correct.